samples/suit/smp_transfer/boards/nrf54h20dk_nrf54h20_cpuapp_common.dtsi

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+ #include "../sysbuild/nrf54h20dk_nrf54h20_memory_map.dtsi"
+
+/ {
+	chosen {
+		zephyr,entropy = &psa_rng;
+		zephyr,uart_mcumgr = &uart136;
```

Comment:
this is wrong, it's `uart-mcumgr` with a dash

Reply:
Fixed